processing-service-core/api/swagger/swagger.yaml

```diff
@@ -1244,16 +1244,20 @@ definitions:
             maxLength: 255
             description: The key of the global config.
   globalConfig:
-    type: object
-    required:
-      - default
-    properties:
-      default:
-        type: string
-        description: The value for the global config key.
-      description:
-        type: string
-        description: The description for the global config key.
+    type: object  
+    required:  
+      - default  
+      - scope  
+    properties:  
+      default:  
+        type: string  
+        description: The value for the global config key.  
+      scope:  
+        type: string  
+        description: Defines the scope of use for the global config key. 
```

Comment:
I think this should be an enum. The contract will currently accept any value for scope but we should be limiting this to only accept currently valid scopes and consider other values as 400 errors. The contract also doesn't indicate what the potential values of scope are and an enum will solve this, both in the written contract and in the generated Java client.